media-libs/libsoundtouch/libsoundtouch-2.1.0.recipe

```diff
@@ -18,23 +18,23 @@ SOURCE_URI="https://gitlab.com/soundtouch/soundtouch/-/archive/$portVersion/soun
 CHECKSUM_SHA256="3fb740c7ac4b304342fbf11a91d14fa3e296269b9fec6266609c4620477d0ad6"
 SOURCE_DIR="soundtouch-$portVersion"
 
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
 commandSuffix=$secondaryArchSuffix
-commandBinDir="$binDir"
+commandBinDir=$binDir
 if [ "$targetArchitecture" = x86_gcc2 ]; then
 	commandSuffix=
-	commandBinDir="$prefix"/bin
+	commandBinDir=$prefix/bin
 fi
 
 libVersion="1.0.0"
 libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
-portVersionCompat="$portVersion compat >= 1"
-
-ARCHITECTURES="x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="x86"
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
```

Comment:
If you want to have "compat >= 2.1" then you could use:
```diff
-portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
+portVersionCompat="$portVersion compat >= ${portVersion%.*}"
```
or hardcode it. I don't know which is best. Well, using
```
portVersionCompat="$portVersion compat >= 2.1"
```
might not be that bad. Indeed, when  `2.2.x` gets released it could be possible that it would be compatible with `2.1.x`.

Reply:
Fixed, I guess by the time they release 2.2.0 we could delete this one after an updated recipe :)